js/wide-tables.jquery.js

```diff
@@ -133,22 +141,24 @@ $.fn.wideTables = function wideTablesPlugin(options) {
     $container.data('id7.wide-tables', wideTables);
   }
 
-  if (!(navigator.userAgent.indexOf('Safari') !== -1 && navigator.userAgent.indexOf('Chrome') === -1)) {
-    if (navigator.platform === 'MacIntel') {
+  if (notSafari()) {
+    if (onMac()) {
       // show scroll bar on non-safari browsers on mac
       // otherwise mac user would not be able to scroll horizontally
       // because the double scroll element might not be visible
       // (and macgic mouse/trackpad horizontal scroll would not work unless on safari.)
       $('.table-responsive').css('overflow-x', 'scroll');
-      $('body').trigger(`non-safari-mac-post-init.${Config.Defaults.namespace}`);
     } else {
       $('.table-responsive').css('overflow-x', 'hidden');
     }
   }
-
   return this.each(attach);
 };
 
+$(window).on(`non-safari-mac-post-init.${Config.Defaults.namespace}`, (e, callback) => {
```

Comment:
I'm not sure about the approach of ID7 listening for an event triggered by the implementing application, then conditionally invoking a callback passed along with the event. This arrangement of code is essentially a function call into ID7. Could we use a different approach?

Reply:
right ok, looking into this 👀

Reply:
previously thought this is more of a tabula case, but thinking about this again I feel it might make more sense to hide double scroll in ID7 on mac when using non-safari as we enabled the system scroll bar already.

Reply:
> previously thought this is more of a tabula case, but thinking about this again I feel it might make more sense to hide double scroll in ID7 on mac when using non-safari as we enabled the system scroll bar already.

right, i've updated so